package.json

```diff
@@ -27,6 +27,7 @@
     "bench": "node bench",
     "lint": "eslint --ignore-path .gitignore .",
     "semantic-release": "semantic-release pre && npm publish && semantic-release post",
+    "pretest": "npx playwright install",
```

Comment:
scripts already have the npm binaries in PATH right? maybe this doesn't need `npx`? or is it this way because we may not have playwright installed as a dependency?

Reply:
You're right, but yes it seems playright isn't a direct devdep so I think this is probably best to keep as is?

Reply:
yup i think so, if it isn't a direct dependency `npx` will ask to install it at least